tests/perf/actor_double_activation/actor_double_activation_test.go

```diff
@@ -2,7 +2,7 @@
 // +build perf
 
 /*
-Copyright 2021 The Dapr Authors
+Copyright 2022 The Dapr Authors
```

Comment:
If you're changing this, then it should be 2023 :)

Reply:
Actually it was wrong because I added this file in 2022 but the license was wrongly pointed to 2021. So I guess I can keep it, right?

Reply:
It doesn't matter, it was just a nit :)